plugin.audio.100fm/addon.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.100fm" name="100FM Radius Digital" version="0.2.0" provider-name="Nir Elbaz">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
Can you please update for me or should I close this request and create a new one?